<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="startStopButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Hosten</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logEditor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Verbinden</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>601</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Senden</string>
   </property>
  </widget>
  <widget class="QListView" name="chatView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>601</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
